src/views/CreateProposal/index.tsx

```diff
@@ -106,66 +142,83 @@ function CreateProposal() {
     setOpenModal(false);
   };
 
+  const handleCloseDaoModal = () => {
+    setOpenDaoModal(false);
+  };
+
   const handleSubmit = async () => {
-      try{
-        await checkLogin()
-        if(wallet.name){
-          const t = await transaction({
-            actions: [{
+    try {
+      await checkLogin();
+      if (wallet.name) {
+        const t = await transaction({
+          actions: [
+            {
               account: "alien.worlds",
-              name: 'transfer',
-              authorization: [{
-                actor: wallet.name,
-                permission: 'active',
-              }],
+              name: "transfer",
+              authorization: [
+                {
+                  actor: wallet.name,
+                  permission: "active",
+                },
+              ],
               data: {
                 from: wallet.name,
                 memo: "deposit",
                 quantity: proposalcost,
-                to: smartcontract
+                to: smartcontract,
               },
-            },{
+            },
+            {
               account: smartcontract,
-              name: 'addproposal',
-              authorization: [{
-                actor: wallet.name,
-                permission: 'active',
-              }],
+              name: "addcrowdf",
+              authorization: [
+                {
+                  actor: wallet.name,
+                  permission: "active",
+                },
+              ],
               data: {
+                wallet: wallet.name,
+                daorules: daorules,
+                title: title,
+                overview: overview,
+                objective: objectives,
                 description: description,
-                memo: memo,
-                title: proposal,
-                tlm: parseFloat(tlmAmount).toFixed(4) + " TLM",
+                business_model: businessModel,
+                duration: duration,
+                teaminfo: teamInfo,
+                requested_funding: parseFloat(tlmAmount).toFixed(4) + " TLM",
                 to: receiveWallet,
-                wallet: wallet.name
+                days_fundraising: daysFundraising,
               },
-            }]
-        })
-        console.log(t)
-        if(t){
-          alert("Succesfully created the proposal")
+            },
+          ],
+        });
+        console.log(t);
+        if (t) {
+          alert("Succesfully created the proposal");
         }
       }
-      }catch(e){
-        alert(e)
-      }
-  }
+    } catch (e) {
+      alert(e);
+    }
+  };
 
-  useEffect(() =>{
-    async function x(){
+  useEffect(() => {
+    async function x() {
       const r = await fetchTable({
-        json: true, 
+        json: true,
         code: smartcontract,
         scope: smartcontract,
         table: "propconfig",
```

Comment:
Use crowdfconfig table and proposal_cost value.

Reply:
Done... Ohhh @!(*&#...  I spend hundreds of tlm for testing 🗡️  My bad...